code/modules/modular_computers/laptop_vendor.dm

```diff
@@ -201,6 +233,10 @@
 			dev_apc_recharger = text2num(href_list["tesla"])
 			fabricate_and_recalc_price(0)
 			return 1
+		if("hw_portable")
+			dev_portable = text2num(href_list["portable"])
+			fabricate_and_recalc_price(0)
+			return 1
```

Comment:
`TRUE`

Reply:
return 1 is actually consistent with the rest of this proc, which either returns 0 or 1 based on success.

TONS of procs do this, such as attack and such. I'd say he's probably fine leaving this as a return 1 for consistency with not just the rest of the proc, but the rest of the codebase until we decided whether or not we want to make return TRUE or return FALSE a standardized styling thing